app/Services/Email.php

```diff
@@ -46,12 +47,15 @@ class Email
     /**
      * Constructor
      */
-    public function __construct(Message $message, Contest $contest, Competition $competition, $users)
+    public function __construct($resources)
     {
-        $this->message = $message;
-        $this->contest = $contest;
-        $this->competition = $competition;
-        $this->users = $users;
+        $this->message = $resources['message'];
+        $this->contest = $resources['contest'];
+        $this->competition = $resources['competition'];
+        $this->users = $resources['users'];
+        // @TODO - is there a better way of instantiating and pulling in the manager class?
+        // Maybe this email class needs to be a service instead.
+        // $this->manager = $manager;
```

Comment:
Yeah, totally makes sense @sbsmith86 :) Just adds a bit of that unavoidable confusion as it gets readjusted!